src/System.CommandLine.Subsystems.Tests/TestData.cs

```diff
@@ -93,4 +93,23 @@ internal class Value : IEnumerable<object[]>
 
         IEnumerator IEnumerable.GetEnumerator() => GetEnumerator();
     }
+
+    internal class Help : IEnumerable<object[]>
+    {
+        // This data only works if the CLI has a --version with a -v alias and also has a -x option
```

Comment:
I do not understand this comment.

Reply:
I believe this is copied from prior test data in this file, e.g. https://github.com/dotnet/command-line-api/blob/240817350b55a9734c0e3d46d1465777c03c8c5b/src/System.CommandLine.Subsystems.Tests/TestData.cs#L17-L23

Reply:
@JeanJoeris is exactly right. Removing this comment in the upcoming commit.